scipy/sparse/tests/test_construct.py

```diff
@@ -37,6 +41,33 @@ def _sprandn_array(m, n, density=0.01, format="coo", dtype=None, random_state=No
 
 
 class TestConstructUtils:
+
+    @pytest.mark.parametrize("cls", [
+        csc_array, csr_array, coo_array, bsr_array,
+        dia_array, dok_array, lil_array
+    ])
+    def test_singleton_array_constructor(self, cls):
+        with pytest.raises(
+            ValueError,
+            match='scipy sparse array classes do not \
+                support instantiation from a scalar'
```

Comment:
The `\` at the end-of-line of the `match` parameter makes pytest look for the indent width of spaces. Instead, close one string, start a new one on a new line and enclose both/all inside parentheses.
```suggestion
            match=('scipy sparse array classes do not '
                          'support instantiation from a scalar'),
```
Or, the `match` keyword supports some basic regular expression parsing. You could use 
`            match='sparse array .* do not support .* a scalar'`